game-core/src/main/java/games/strategy/triplea/oddsCalculator/ta/AggregateResults.java

```diff
@@ -12,27 +12,31 @@
 import games.strategy.triplea.util.TuvUtils;
 import games.strategy.util.IntegerMap;
 import games.strategy.util.Tuple;
+import lombok.Getter;
+import lombok.Setter;
 
 public class AggregateResults implements Serializable {
```

Comment:
Notable concern. `oddsCalculator` is battle calc, generally should not be saved to disk or sent over network. 

I'm pretty sure serialization here is because of threaded'ness. The idea being that if your thread was parked it could be serialized.. At one time that was good practice. To be honest I'm not 100% sure why it is serializable. For what it's worth I checked a network game with battle calc and loading save game, they checked out.

Reply:
> oddsCalculator is battle calc, generally should not be saved to disk or sent over network

Don't let that necessarily give you a warm, fuzzy feeling: #2208 :smile:

Reply:
Indeed. I've double checked save game compat, it looks good.